sleepecg/test/test_examples.py

```diff
@@ -0,0 +1,32 @@
+# Authors: Florian Hofer
+#
+# License: BSD (3-clause)
+
+"""Tests to make sure examples don't crash."""
+
+import runpy
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+
+EXCLUDE = [
+    '*/benchmark/*',
+]
+
+examples_dir = (Path(__file__).parent / '../../examples').resolve()
+
+files_to_test = []
+for pyfile in examples_dir.rglob('*.py'):
+    for pattern in EXCLUDE:
+        if not pyfile.match(pattern):
+            files_to_test.append(pyfile)
```

Comment:
The logic doesn't work out - if a file does not match the first pattern you are already adding it to the files to test. But what if it matches another pattern defined in `EXCLUDE`?

Reply:
True, fixed!